angular.json

```diff
@@ -247,9 +247,80 @@
         "test": {
           "builder": "@angular-devkit/build-angular:karma",
           "options": {
-            "main": "src/clr-base/src/test.ts",
+            "main": "src/clr-base/test.ts",
             "tsConfig": "src/clr-base/tsconfig.spec.json",
-            "karmaConfig": "src/clr-base/karma.conf.js"
+            "karmaConfig": "src/clr-base/karma.conf.js",
+            "codeCoverageExclude": ["src/clr-base/common/css-vars/css-vars-ponyfill-fork.ts"]
+          }
+        }
+      }
+    },
+    "dev-base": {
+      "projectType": "application",
+      "schematics": {
+        "@schematics/angular:component": {
+          "style": "scss"
+        }
+      },
+      "root": "src/dev-base",
+      "sourceRoot": "src/dev-base/src",
+      "prefix": "app",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
```

Comment:
Shouldn't this use `@angular-builders/custom-webpack:browser` and the `custom-webpack.config` that we had to use in the dev-app to get it working in IE?

Reply:
I need to test that the non traspiled es5 code/modules run properly so I need to figure out a way to configure this separately or have a ie specific build. Also the es5 transpile breaks the unit testing.